src/org/loklak/harvester/strategy/KaizenHarvester.java

```diff
@@ -94,6 +109,17 @@ private void grabInformation(Timeline timeline) {
             if (!place.isEmpty())
                addQuery("near:\"" + message.getPlaceName() + "\" within:" + PLACE_RADIUS + "mi");
         }
+
+        if (query != null && oldestTweetDate != null) {
+            String oldestTweetDateStr = dateToString.format(oldestTweetDate);
+            int startIndex = query.indexOf("until:");
```

Comment:
Because once we have the oldest Tweet in the current timeline, we would like to get even older Tweets. So, we need to get the Tweets that were posted earlier than this date, so I made it until.

`until` - Tweets which were created before Apr 22, 2017.
`since` - Tweets which were created after (since) Apr 22, 2017.